app/controllers/unattended_controller.rb

```diff
@@ -11,14 +11,16 @@ class UnattendedController < ApplicationController
   # We want to find out our requesting host
   before_action :get_host_details, :except => [:hostgroup_template, :built, :failed]
   before_action :get_built_host_details, :only => [:built, :failed]
-  before_action :allowed_to_install?, :except => :hostgroup_template
+  before_action :allowed_to_install?, :except => :hostgroup_template, :unless => -> { ipxe_request? }
```

Comment:
This is hacky, can you move the logic into `allowed_to_install?` method itself?

Also, this opens doors to all iPXE requests while we only want to allow the initial request. Can we identify the initial (bootstrap) request? How about passing an extra option from DHCP filename option that would do this? Something like `/unattended/iPXE?bootstrap=yes` or we can reuse `token=boostrap` perhaps.